assignment5.txt

```diff
@@ -0,0 +1,23 @@
+Olivier Gabison
+
+1. I would call it by doing printf("%d", fxn(1, 2.0, 3));
```

Comment:
Right, don't forget that a declaration is needed before the main or it will return a warning.